installation/local/bootstrap.sh

```diff
@@ -113,12 +131,12 @@ cmake \
     -DPYTHON_INCLUDE_DIR=/usr/include/python3.10 \
     -DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.10.so
 
-make -j 3
+make -j 4
```

Comment:
Maybe this should be a variable as well?
You bumped 3 to 4 here but for Insite it is still 3.
Is this on purpose ?

Reply:
I will change it. overlooked.
If everything else is fine, we merge